moveit_core/collision_distance_field/src/collision_distance_field_types.cpp

```diff
@@ -452,9 +453,9 @@ void collision_detection::getProximityGradientMarkers(
   {
     for (unsigned int j = 0; j < gradients[i].distances.size(); j++)
     {
-      visualization_msgs::Marker arrow_mark;
+      visualization_msgs::msg::Marker arrow_mark;
       arrow_mark.header.frame_id = frame_id;
-      arrow_mark.header.stamp = ros::Time::now();
+      arrow_mark.header.stamp = ros_clock.now();
```

Comment:
http://docs.ros2.org/bouncy/api/rclcpp/classrclcpp_1_1_clock.html

This needs to be RCL_ROS_TIME not RCL_SYSTEM_TIME which is the default

Reply:
https://github.com/ros-planning/moveit2/pull/65/commits/65137dba2012aef65d4fd181c21c54f4af2bfed0